### ISSUE TITLE
Double coloring resulting in garbage when using `list` in Python 3

### ISSUE THREAD

**Issue by antocuni** [pdbpp/pdbpp#35]
Originally reported by: **Marc Abramowitz (Bitbucket: [msabramo](https://bitbucket.org/msabramo), GitHub: [msabramo](https://github.com/msabramo))**

----------------------------------------

See screenshot:

![Screen Shot 2014-12-08 at 6.35.38 AM.png](https://bitbucket.org/repo/Grbzr/images/2646570589-Screen%20Shot%202014-12-08%20at%206.35.38%20AM.png)

```
❯ python -V
Python 3.3.3

❯ pip freeze
Pygments==2.0.1
fancycompleter==0.4
ipdb==0.8
ipython==1.2.1
-e hg+ssh://hg@bitbucket.org/antocuni/pdb@4669c3747a396e3766173feb40ebece32ab08aad#egg=pdbpp-dev
-e hg+ssh://hg@bitbucket.org/pypy/pyrepl@cef132a7e4649e84aa26a2b1af1f5a1703c001a0#egg=pyrepl-dev
wmctrl==0.1
```

----------------------------------------
- Bitbucket: https://bitbucket.org/antocuni/pdb/issue/35

**Comment by antocuni** [comment]
*Original comment by* **Marc Abramowitz (Bitbucket: [msabramo](https://bitbucket.org/msabramo), GitHub: [msabramo](https://github.com/msabramo))**:

----------------------------------------

BB-35: Fix double-coloring of list in Python 3

Fixes issue #35

**Comment by antocuni** [comment]
*Original comment by* **Marc Abramowitz (Bitbucket: [msabramo](https://bitbucket.org/msabramo), GitHub: [msabramo](https://github.com/msabramo))**:

----------------------------------------

See https://bitbucket.org/antocuni/pdb/pull-request/26/bb-35-fix-double-coloring-of-list-in/diff

**Comment by antocuni** [comment]
*Original comment by* **Marc Abramowitz (Bitbucket: [msabramo](https://bitbucket.org/msabramo), GitHub: [msabramo](https://github.com/msabramo))**:

----------------------------------------

Reopening - my fix was not a good one; see this instead: https://bitbucket.org/antocuni/pdb/pull-request/26/bb-35-fix-double-coloring-of-list-in/diff#comment-5155943

**Comment by antocuni** [comment]
*Original comment by* **Marc Abramowitz (Bitbucket: [msabramo](https://bitbucket.org/msabramo), GitHub: [msabramo](https://github.com/msabramo))**:

----------------------------------------

Fix double-coloring by rename _print_lines => _print_lines_pdbpp

As @antocuni said in https://bitbucket.org/antocuni/pdb/pull-request/26/bb-35-fix-double-coloring-of-list-in/diff#comment-5155943:

> I think it's just a name clash: _print_lines is an internal method introduced
> by pdb++ which was not there in 2.7's pdb.py. By chance, 3.4's pdb.py
> introduced its own _print_lines. Probably, we can fix it by simply rename our
> _print_lines to something else (e.g. _print_many_lines); this should also
> kill the need for the ugly _py3_print_lines vs _py2_print_lines hack

Fixes #35
